ddtrace/settings/config.py

```diff
@@ -207,10 +457,6 @@ def __init__(self):
         self.tags = gitmetadata.clean_tags(parse_tags_str(os.getenv("DD_TAGS") or ""))
 
         self.env = os.getenv("DD_ENV") or self.tags.get("env")
```

Comment:
Is this PR only adding support for config sources? I'm assuming `env, version` also should be ported over, but is not in the scope of this PR?